opencv_flask/app.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, request
+import cv2
+import numpy as np
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+@app.route('/display')
+def display_vid():
+    file_name = request.args.get('file_name', default = '', type = str)
+    cap = cv2.VideoCapture(file_name)
+    if (cap.isOpened()== False): 
+        print("Error opening video stream or file")
```

Comment:
Give some kind of information on where this error is coming from. Judging by this if statement, it seems like it's coming from cv2

Reply:
added. cv2 is opencv.